.github/workflows/create-cherry-pick-pr.yml

```diff
@@ -13,19 +13,18 @@ on:
         description: 'PR # Associated with Cherry Pick'
         required: true
 
-
 jobs:
   cherry-pick:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v4
+      - name: Checkout code
+        uses: actions/checkout@v4
 
-    - name: Create Cherry Pick PR
-      id: create-cherry-pick-pr
-      shell: bash
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: |
-        ./scripts/create-cherry-pick-pr.sh ${{ github.event.inputs.branch_name }} ${{ github.event.inputs.commit_hash }} ${{ github.event.inputs.PR_number }}
\ No newline at end of file
+      - name: Create Cherry Pick PR
+        id: create-cherry-pick-pr
+        shell: bash
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          ./scripts/create-cherry-pick-pr.sh ${{ github.event.inputs.branch_name }} ${{ github.event.inputs.commit_hash }} ${{ github.event.inputs.PR_number }}
```

Comment:
Is this path relative to this `workflows` directory or the `.github` directory?

If the former, would it need to be `../scripts`?

Reply:
Good point, looking at the mobile repo implementation (which inspired this one and is currently working), this path is relative to the repo root folder. I still think keeping it in the `.github` folder makes sense, so I'm changing the path accordingly.

https://github.com/MetaMask/metamask-mobile/blob/main/.github/workflows/create-cherry-pick-pr.yml#L38-L39
https://github.com/MetaMask/metamask-mobile/blob/main/scripts/create-cherry-pick-pr.sh